source/dump.f90

```diff
@@ -818,31 +876,45 @@ subroutine dump_initialise
+            "<px*y> <px*py> <px*sigma> <px*psigma> <y^2> <y*py> <y*sigma> <y*psigma> <py^2> <py*sigma> <py*psigma> <sigma^2> "//   &
+            "<sigma*psigma> <psigma^2>",colNames,nSplit,spErr)
+          call chr_split("1 1 m m 1 m 1 m 1 mm^2 mm mm^2 mm mm^2 mm 1 mm 1 mm 1 mm^2 mm mm^2 mm 1 mm 1 mm^2 mm 1",&
+            colUnits,nSplit,spErr)
+          call h5_writeDataSetAttr(dump_hdf5DataSet(i),"dumpFormat",6)
+          call h5_writeDataSetAttr(dump_hdf5DataSet(i),"colNames",  colNames)
+          call h5_writeDataSetAttr(dump_hdf5DataSet(i),"colUnits",  colUnits)
+        end block
 
       case(101)
         ! Format 101:
         ! # particleID turn s[m] x[mm] xp[mrad] y[mm] yp[mrad] z[mm] (E-E0)/E0[1] ktrack
         if(dump_hdf5Format(3) == 0) then
-          allocate(setFields(10))
-          setFields(1)  = h5_dataField(name="ID",     type=h5_typeInt)
-          setFields(2)  = h5_dataField(name="TURN",   type=h5_typeInt)
-          setFields(3)  = h5_dataField(name="S",      type=h5_typeReal)
-          setFields(4)  = h5_dataField(name="X",      type=h5_typeReal)
-          setFields(5)  = h5_dataField(name="XP",     type=h5_typeReal)
-          setFields(6)  = h5_dataField(name="Y",      type=h5_typeReal)
-          setFields(7)  = h5_dataField(name="YP",     type=h5_typeReal)
-          setFields(8)  = h5_dataField(name="dE/E",   type=h5_typeReal)
-          setFields(9)  = h5_dataField(name="Z",      type=h5_typeReal)
-          setFields(10) = h5_dataField(name="KTRACK", type=h5_typeInt)
-          setFields(11)  = h5_dataField(name="E",      type=h5_typeReal)
-          setFields(12)  = h5_dataField(name="PC",      type=h5_typeReal)
-          setFields(13)  = h5_dataField(name="P/P0",     type=h5_typeReal)
-          setFields(14)  = h5_dataField(name="P0/P)",      type=h5_typeReal)
-          setFields(15)  = h5_dataField(name="BETA0/BETA",     type=h5_typeReal)
-          setFields(16)  = h5_dataField(name="MASS",   type=h5_typeReal)
-          setFields(17)  = h5_dataField(name="M/M0/Q0/Q", type=h5_typeReal)
-          setFields(18)  = h5_dataField(name="ENERGY0",      type=h5_typeReal)
-          setFields(19)  = h5_dataField(name="PC0",      type=h5_typeReal)
+          allocate(setFields(19))
+          setFields(1)  = h5_dataField(name="ID",         type=h5_typeInt)
+          setFields(2)  = h5_dataField(name="TURN",       type=h5_typeInt)
+          setFields(3)  = h5_dataField(name="S",          type=h5_typeReal)
+          setFields(4)  = h5_dataField(name="X",          type=h5_typeReal)
+          setFields(5)  = h5_dataField(name="XP",         type=h5_typeReal)
+          setFields(6)  = h5_dataField(name="Y",          type=h5_typeReal)
+          setFields(7)  = h5_dataField(name="YP",         type=h5_typeReal)
+          setFields(8)  = h5_dataField(name="dE/E",       type=h5_typeReal)
+          setFields(9)  = h5_dataField(name="Z",          type=h5_typeReal)
```

Comment:
sigma vs. z -- is this correctly done here?

Reply:
This is Riccardo's code, but I changed this to SIGMA.